pages/download.html

```diff
@@ -152,7 +154,36 @@ <h2><img src="{% static '/static/images/download_ubuntu.png' %}" class="feature_
     <p><a href="{{ unstable_ubuntu64 }}"
           onclick="javascript:trackDownload('{{ unstable_ubuntu64_analytics_conversion }}');">
         {% blocktrans %}{{ unstable_version }} for 64-bit Ubuntu Bionic{% endblocktrans %}</a></p>
+  </div>
+
+  <div class="halfbox_right darkborder fedorabox">
+    <h2><img src="{% static '/static/images/download_fedora.png' %}" class="feature_icon">{{ fedora }}</h2>
+
+    <p>{% blocktrans %}
+        An <a href="https://admin.rpmfusion.org/pkgdb/package/free/mixxx/">RPM package</a>
+        for installation is available in the
+        <a href="https://rpmfusion.org">RPM Fusion</a> repositories.
+      Please refer to their instructions on how to enable the repositories on your system.
```

Comment:
```suggestion
      Please refer to their <a href="https://rpmfusion.org/Configuration">instructions</a> on how to enable the repositories on your system.
```

Reply:
Add: "Mixxx only requires enabling the free repository; the nonfree repository is not necessary for Mixxx."